specs-go/v1/config.go

```diff
@@ -75,6 +75,7 @@ type History struct {
 }
 
 // Image is the JSON structure which describes some basic information about the image.
+// This provides the `application/vnd.oci.image.config.v1+json` mediatype when marshalled to JSON.
```

Comment:
i was thinking the same. Especially since the field is omitEmpty for JSON. Perhaps make it *optional* with a SHOULD, so it will have compat with older docker? Thoughts @stevvooe ?